src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,69 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] inputArray = readingFromFile(fromFileName).split(",");
+        writingInToNewFile(toFileName, inputArray);
```

Comment:
The name suggests that you're trying to write an inputArray to file, but you're actually doing some operations on this file and then write results. Don't apply side effects to your code.